packages/symplify-kernel/composer.json

```diff
@@ -7,9 +7,9 @@
         "symfony/console": "^6.2",
         "symfony/dependency-injection": "6.1.*",
         "symplify/smart-file-system": "^11.2",
-        "symplify/autowire-array-parameter": "^11.2",
         "symplify/package-builder": "^11.2",
-        "webmozart/assert": "^1.11"
+        "webmozart/assert": "^1.11",
+        "symplify/autowire-array-parameter": "^11.1.25"
```

Comment:
@TomasVotruba this is strange, I update symplify-kernel, but the split is not applied to symplify-kernel https://github.com/symplify/symplify-kernel , could you verify? 

see https://github.com/symplify/symplify/actions/runs/4074326681/jobs/7019339731#step:4:3

This update of symplify-kernel is required by its autowire-array-parameter